app/config.js

```diff
@@ -3,12 +3,14 @@
 angular.module('config', [])
   .constant('config', {
     'endpoint'    : "http://"+window.location.hostname+":8080",
-    'provider'    : "basic", // google, github, gitlab, keycloak, saml2 or basic
+    'provider'    : "basic", // google, github, gitlab, keycloak, pingfederate, saml2 or basic
     'client_id'   : "INSERT-CLIENT-ID-HERE",
     'github_url'  : null,  // replace with your enterprise github server
     'gitlab_url'  : "https://gitlab.com",  // replace with your gitlab server
     'keycloak_url': "https://keycloak.example.org",  // replace with your keycloak server
     'keycloak_realm': "master",  // replace with your keycloak realm
+    'pf_oauth_url' : "https://pingfederate.example.org" // replace with your pingfederate server
+    'redirectUri'  : "http://alerta.example.com/" // replace with your redirect url registered in pingfederate
```

Comment:
I don't think this config variable is needed. Just use `window.location.orgin` like the other auth configs.

Reply:
in the ping federate redirection where we were needed to configure the redirect url it was needed to have url end with `/` but window.location.origin was not having that hence the redirection wasn't working